salt/config/__init__.py

```diff
@@ -2787,7 +2787,7 @@ def cloud_config(path, env_var='SALT_CLOUD_CONFIG', defaults=None,
     apply_sdb(opts)
 
     # prepend root_dir
-    prepend_root_dirs = ['cachedir']
+    prepend_root_dirs = ['cachedir','log_file']
     prepend_root_dir(opts, prepend_root_dirs)
```

Comment:
@samilaine but can we maybe not to call this list "prepend_root_dirs", since it is path options? I understand that the variable was already there, but let's agree it is greatly misleading name. Maybe I would just do it so:

```python
prepend_root_dir(opts, ['cachedir','log_file'])
```

So whoever wants to know how this is working, can lookup the function and see that this is just a `path_options`.

Reply:
However, it turns out that there might be a reasonable explanation for that funny variable `prepend_root_dirs` (see line [2358](https://github.com/saltstack/salt/blob/develop/salt/config/__init__.py#L2538) in `salt/config/__init__.py`) .

So, instead of getting rid of the funny variable, I opted for abusing it even more.

Reply:
for the record, i am for leaving the variable, as it is how all the other config functions work, master_config, minion_config, etc.

Reply:
@samilaine :wink: Well, OK. I approved this anyway, and that one was up to you. Thanks!